STACK-UP-3/champs-frontend
# 173169085-oneway-trip :create oneway trip

#### Description
This PR adds create oneWay trip functionality  from the UI to the codebase
#### Type of change

Please select the relevant option


#### How Has This Been Tested?



#### How to test?

> - Clone the branch locally
> - Run `npm start`
> - Go to your browser and go to `localhost:8080/home` **NB:**  log in if you haven't done it yet
> - click `create trip` button
> - Insert required information then you will see a toast telling you that the trip was successfully created or if there was an error

#### Screnshoot?


![1](https://user-images.githubusercontent.com/60552218/85324171-a3ef0c00-b4c9-11ea-88ac-eb6912dc6c22.jpg)
![3](https://user-images.githubusercontent.com/60552218/85324194-ab161a00-b4c9-11ea-8dc0-6defa0fcb18f.jpg)

#### STORY ID

[Finishes #173169085]
